services/crypto_delete_allowance.proto

```diff
@@ -0,0 +1,95 @@
+syntax = "proto3";
+
+package proto;
+
+/*-
+ * ‌
+ * Hedera Network Services Protobuf
+ * ​
+ * Copyright (C) 2018 - 2022 Hedera Hashgraph, LLC
+ * ​
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * ‍
+ */
+
+option java_package = "com.hederahashgraph.api.proto.java";
+option java_multiple_files = true;
+
+import "basic_types.proto";
+
+/**
+ * Deletes one or more hbar/token approved allowances from an owner's account. This operation
+ * will remove the allowances granted to <b>all</b> spenders of an owner's hbar, a specific
+ * fungible token and/or a specific non-fungible token serial number.
```

Comment:
We might add a comment on the signing requirements of the transaction,
```
Each owner account listed as wiping an allowance must sign the transaction.
```

Reply:
Fixed !